packages/graphiql/src/css/app.css

```diff
@@ -496,6 +496,16 @@ div.CodeMirror-lint-tooltip > * + * {
   margin-top: 12px;
 }
 
+.graphiql-container .variable-editor-title-text {
+  cursor: pointer;
+  display: inline-block;
+  color: gray;
+}
+
+.graphiql-container .variable-editor-title-text.active {
```

Comment:
I could not identify a pattern for the naming of such selectors. Happy to replace `.active` with something else if desired!

Reply:
Oh, that’s should be fine!